src/components/ConfigProvider/ParentComponentsContext.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { createContext, FC, useContext, useMemo } from 'react';
```

Comment:
If this is to be a abstracted context that may be used by any parent, Move this into https://github.com/EightfoldAI/octuple/tree/main/src/components/ConfigProvider

Reply:
Exporting this and its props may also be helpful.